FluentAssertions.Core/Types/MethodInfoAssertions.cs

```diff
@@ -86,19 +94,14 @@ public MethodInfoAssertions(MethodInfo methodInfo)
         /// <param name="reasonArgs">
         /// Zero or more objects to format using the placeholders in <see cref="because" />.
         /// </param>
-        public AndConstraint<MethodInfoAssertions> BeVirtual(
-            string because = "", params object[] reasonArgs)
+        public AndConstraint<MethodBaseAssertions<MethodInfo, MethodInfoAssertions>> Return(Type returnType, string because = "", params object[] reasonArgs)
```

Comment:
Done. In fact, I added overloads with Generic Parameters to all assertions that took fixed numbers of type parameters.